src/azure-cli/azure/cli/command_modules/acs/_params.py

```diff
@@ -211,24 +214,25 @@ def load_arguments(self, _):
         c.argument('name', validator=validate_linux_host_name)
         c.argument('kubernetes_version',
                    completer=get_k8s_versions_completion_list)
+        c.argument('dns_name_prefix', options_list=['--dns-name-prefix', '-p'])
+        c.argument('node_osdisk_diskencryptionset_id', options_list=['--node-osdisk-diskencryptionset-id', '-d'])
+        c.argument('disable_local_accounts', action='store_true')
+        c.argument('disable_rbac', action='store_true')
+        c.argument('enable_rbac', action='store_true', options_list=['--enable-rbac', '-r'],
+                   deprecate_info=c.deprecate(redirect="--disable-rbac", hide="2.0.45"))
```

Comment:
> hide="2.0.45"

Why show it before version `2.0.45`?

Reply:
Sure, deleted in commit [#95637a3](https://github.com/Azure/azure-cli/pull/22165/commits/95637a39cbed256c95b023cd22ddd43c423b948d).